perfkitbenchmarker/benchmark_sets.py

```diff
@@ -71,7 +71,10 @@
     'google_set': {
         MESSAGE: ('This benchmark set is maintained by Google Cloud Platform '
                   'Performance Team.'),
-        BENCHMARK_LIST: [STANDARD_SET]
+        BENCHMARK_LIST: [STANDARD_SET,
+                         'block_storage_logging',
+                         'block_storage_database',
+                         'block_storage_streaming']
```

Comment:
`block_storage_workload` is still in the standard set, so this seems duplicative.